internal/services/regional_hostname/migrations.go

```diff
@@ -6,10 +6,83 @@ import (
 	"context"
 
 	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
 )
 
 var _ resource.ResourceWithUpgradeState = (*RegionalHostnameResource)(nil)
 
 func (r *RegionalHostnameResource) UpgradeState(ctx context.Context) map[int64]resource.StateUpgrader {
-	return map[int64]resource.StateUpgrader{}
+	return map[int64]resource.StateUpgrader{
```

Comment:
Is it possible to have tests for this?
Non-blocking -- can be in another PR if its easier

Reply:
yeah, although not sure what coverage is missing from `TestAccCloudflareRegionalHostname_Migration_TimeoutsRemoval`  that should smoke test this right?

Reply:
moved the test for this into a migrations_test.go in 7807eda